ansible/idr-omero-readonly.yml

```diff
@@ -185,4 +185,23 @@
   roles:
   - role: openmicroscopy.omero-server
 
+  tasks:
+    - name: Override Bio-Formats artifacts in lib/server
```

Comment:
Minor nitpick, can you add an empty line between multiple tasks in this and other playbooks?

Reply:
Done in the separate playbook